content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,191 @@
+sections are reviewed.]
+
+
+## Improvements that graduated to stable in Kubernetes v1.30 {#graduations-to-stable}
+
+_This is a selection of some of the improvements that are now stable following the v1.30 release._
+
+### Robust VolumeManager reconstruction after kubelet restart ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+This is a volume manager refactoring that allows the kubelet to populate additional information about how existing volumes are mounted during the kubelet startup. In general, this makes volume cleanup after kubelet restart or machine reboot more robust.
+
+This does not bring any changes for user or cluster administrators. We used the feature process and feature gate `NewVolumeManagerReconstruction` to be able to fall back to the previous behavior in case something goes wrong. Now that the feature is stable, the feature gate is locked and cannot be disabled.
+
+### Prevent unauthorized volume mode conversion during volume restore ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+With the release of Kubernetes 1.30, the ability to inherently prevent unauthorized users from converting the volume mode of a PVC, when it is created from an existing VolumeSnapshot, moves to GA. Cluster administrators will now have to specifically grant permissions in Kubernetes to authorized users, like storage backup vendors, to allow them to perform such actions. 
+For more information, see [converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode).
+
+### Pod Scheduling Readiness ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3521 goes here.
+
+### Min domains in PodTopologySpread ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3022 goes here.
+
+### Go workspaces for k/k ([SIG Architecture](https://github.com/kubernetes/community/tree/master/sig-architecture))
+
+The Kubernetes repo now uses Go workspaces. This should not impact end users at all, but does have a impact for developers of downstream projects. Switching to workspaces caused some breaking changes in the flags to the various [k8s.io/code-generator](https://github.com/kubernetes/code-generator) tools. Downstream consumers should look at [`staging/src/k8s.io/code-generator/kube_codegen.sh`](https://github.com/kubernetes/code-generator/blob/master/kube_codegen.sh) to see the changes.
+
+For full details on the changes and reasons why Go workspaces was introduced, read [Using Go workspaces in Kubernetes](https://www.kubernetes.dev/blog/2024/03/19/go-workspaces-in-kubernetes/).
+
+## Improvements that graduated to beta in Kubernetes v1.30 {#graduations-to-beta}
+
+_This is a selection of some of the improvements that are now beta following the v1.30 release._
+
+### Node log query ([SIG Windows](https://github.com/kubernetes/community/tree/master/sig-windows))
```

Comment:
```suggestion
### [Node log query] (/docs/concepts/cluster-administration/system-logs/#log-query)([SIG Windows](https://github.com/kubernetes/community/tree/master/sig-windows))
```
Based on @sftim & @dipesh-rawat  suggestions

Reply:
I'm going to go with Abbie link suggestion from our Slack conversation